04.ls/ls.rb

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+files = Dir.glob('*').sort
+
+cols = 3
+max_row = (files.length / cols.to_f).ceil
+max_char_length = files.map(&:length).max
+print_files = Array.new(max_row) { Array.new(cols) }
```

Comment:
ファイルをプリントする、というメソッドのように見えてしまいます
https://qiita.com/jnchito/items/459d58ba652bf4763820

また、配列の初期化ですが、colsの方は`[]`だけにしてもいいかもしれません。

```suggestion
print_files = Array.new(max_row) { [] }
```